webstorage/idlharness.html

```diff
@@ -7,7 +7,7 @@
 <link rel="help" href="http://www.w3.org/TR/webstorage/#storage"/>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/webidl2/lib/webidl2.js"></script>
```

Comment:
Hmm, seems the current being http://w3c-test.org/ doesn't use `WebIDLParser.js` name; shall we update all idlharness.js based test files?